go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/clastix/capsule v0.1.0
 	github.com/containernetworking/plugins v0.9.1
 	github.com/coreos/go-iptables v0.5.0
+	github.com/docker/docker v20.10.5+incompatible
```

Comment:
Would personally avoid to depend on the entire docker just to generate a random name, but that is a matter of opinion. Could leverage github.com/dustinkirkland/golang-petname as a standalone library